unix/modtime.c

```diff
@@ -152,6 +152,9 @@ STATIC const mp_rom_map_elem_t mp_module_time_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_time), MP_ROM_PTR(&mod_time_time_obj) },
     { MP_ROM_QSTR(MP_QSTR_ticks_ms), MP_ROM_PTR(&mp_utime_ticks_ms_obj) },
     { MP_ROM_QSTR(MP_QSTR_ticks_us), MP_ROM_PTR(&mp_utime_ticks_us_obj) },
+#ifdef _WIN32
+    { MP_ROM_QSTR(MP_QSTR_ticks_cpu), MP_ROM_PTR(&mp_utime_ticks_cpu_obj) },
+#endif
```

Comment:
To reduce the number of #if's we should just have this line added for both unix and windows, ie without the #ifdef.  It'll still compile but on unix ticks_cpu() will return 0.  That's fine for now, it can be implemented later.